splash/qtrender_image.py

```diff
@@ -68,20 +77,20 @@ def is_jpeg(self):
     def is_png(self):
         return self.image_format == 'PNG'
 
-    def qimage_to_pil_image(self, qimage):
+    def _qimage_to_pil_image(self, qimage: QImage) -> Image:
```

Comment:
The travis tests are failed with
```
E   AssertionError: 400 != 200 : (400, 
'{"info": {"message": "Lua error: AttributeError(\\"\'QtImageRenderer\' object has no attribute 
\'qimage_to_pil_image\'\\",)", "type": "LUA_ERROR"}, "description": "Error happened while executing Lua script", 
"error": 400, "type": "ScriptError"}',
 'http://localhost:44451/execute?url=http%3A%2F%2Flocalhost%3A49991%2Fvery-long-green-page&viewport=50x1024&render_all=1&engine=webkit&lua_source=main+%3D+require%28%22emulation%22%29.render_png&wait=0.01')
```

https://travis-ci.org/scrapinghub/splash/builds/496767326

Reply:
see #889